apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-queue-management-autoselection.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { createFakeVisitor } from '../../mocks/data';
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { HomeOmnichannel, OmnichannelLiveChat } from '../page-objects';
+import { test, expect } from '../utils/test';
+
+const firstVisitor = createFakeVisitor();
+
+const secondVisitor = createFakeVisitor();
+
+test.use({ storageState: Users.user1.state });
+
+test.describe('OC - Livechat - Queue Management', () => {
+	test.skip(!IS_EE, 'Enterprise Only');
+
+	let poHomeOmnichannel: HomeOmnichannel;
+	let poLiveChat: OmnichannelLiveChat;
+
+	const waitingQueueMessage = 'This is a message from Waiting Queue';
+	const queuePosition1 = 'Your spot is #1';
+	const queuePosition2 = 'Your spot is #2';
+
+	test.beforeAll(async ({ api, browser }) => {
+		await Promise.all([
+			api.post('/settings/Livechat_Routing_Method', { value: 'Auto_Selection' }),
+			api.post('/settings/Livechat_accept_chats_with_no_agents', { value: true }),
+			api.post('/settings/Livechat_waiting_queue', { value: true }),
+			api.post('/settings/Livechat_waiting_queue_message', { value: waitingQueueMessage }),
+			api.post('/livechat/users/agent', { username: 'user1' }),
+		]);
+
+		const { page: omniPage } = await createAuxContext(browser, Users.user1, '/', true);
+		poHomeOmnichannel = new HomeOmnichannel(omniPage);
+
+		// Agent will be offline for these tests
+		await poHomeOmnichannel.sidenav.switchOmnichannelStatus('offline');
+	});
+
+	test.beforeEach(async ({ browser, api }) => {
+		const context = await browser.newContext();
+		const page2 = await context.newPage();
+
+		poLiveChat = new OmnichannelLiveChat(page2, api);
+		await poLiveChat.page.goto('/livechat');
+	});
+
+	test.afterAll(async ({ api }) => {
+		await Promise.all([
+			api.post('/settings/Livechat_waiting_queue', { value: false }),
+			api.post('/settings/Livechat_waiting_queue_message', { value: '' }),
```

Comment:
Yup, that's intentional. `Auto_Selection` is the default value for the Routing algo, since we set it on the before, didn't change again on the after as it would be the same.